README.md

```diff
@@ -1,7 +1,7 @@
 ![](https://runelite.net/img/logo.png)
 # plugin-hub [![Discord](https://img.shields.io/discord/301497432909414422.svg)](https://discord.gg/mePCs8U)
 
-This repository contains markers for [RuneLite](https://github.com/runelite/runelite)
+This repositorya contains markers for [RuneLite](https://github.com/runelite/runelite)
```

Comment:
Accidental change?

Reply:
@adamk33n3r  My changes were not showing up on github even after I pushed until a day later, I could change this back but it was intentional.

Reply:
To clarify I was pushing changes, even force pushing, and this PR was not updating, that's why I did this, seems now it's instantly updating 🤷

Reply:
Oh, weird. Never seen that before.